### PR TITLE
Update Emogotchi live URL

### PR DESCRIPTION
**Pull request by RajithaKumara**
Should change the Emogotchi live URL because there was no repository under https://github.com/mdn/emogotchi

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
Ah yes, good point. Thanks!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # emogotchi
 A simple parody of Tamagotchi, in which you actively have to help your Emo stay miserable to avoid losing the game. Written to demonstrate the [W3C Notifications API](https://developer.mozilla.org/en-US/docs/Web/API/Notifications_API).
```
Comment (Ahapulin) [inline README.md:3]: 34f43ebf5282390acb067792dd240f377a381f9b

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
```
Comment (Ahapulin) [inline README.md:2]: 34f43ebf5282390acb067792dd240f377a381f9b

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
 
-You can [view the Emogotchi live](http://mdn.github.io/emogotchi/).
```
Comment (Ahapulin) [inline README.md:4]: 34f43ebf5282390acb067792dd240f377a381f9b

**Review by Ahapulin — APPROVED** [review]

**Pull request merged**